src/test/java/io/github/opensanca/DemoSpringBootWebApp.java

```diff
@@ -1,9 +1,11 @@
 package io.github.opensanca;
 
+import java.util.Locale;
```

Comment:
As a side note, we need to standardize our editor preferences so the diffs doesn't get polluted with minor things like imports being reordered or other formatting issues.